documentation/docs/steps/apiKeyValueMapUpload.md

```diff
@@ -0,0 +1,47 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+With this step you can store one or more key value pairs of data stored in a group called a map or key value map.<br />
+To consume the ApiKeyValueMapUpload step, proceed as follows:
+
+* Get the SAP API management service key from the SAP BTP sub account cockpit, under instance and subscriptions for service API Management, API portal under API plan.
+* Store your service key created for SAP API Management in the the Jenkins server as a secret text.
+* Create a new Jenkins pipeline designated for the ApiKeyValueMapUpload step.
+* Execute the pipeline and validate step exection results as explained in the blog [Integration Suite Piper commands](https://blogs.sap.com/2022/01/05/orking-with-integration-suite-piper-commands/)
+The ApiKeyValueMapUpload step allows you to:
+
+* You can create new API key value map in the API portal.
+* Prevent command execution in case the/an API key value map already exists.
```

Comment:
Is this true?

Reply:
can you shared generated document to see how it looks?

Reply:
@rodibrin Could you build and share the latest changes?

Reply:
> Please find one writing style. This bullet point is a half sentence that does not make sense without the title. So the full sentence would be: "The ApiKeyValueMapUpload step allows you to prevent command execution in case the/an API key value map already exists." While the other bullet points are full sentences: "You can create new API key value map in the API portal."
> 
> Choose one style and stick to it, please. It is making it much more difficult to read otherwise.

This was not fixed!